extensions/amp-social-share/1.0/test/test-social-share.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {SocialShare} from '../social-share';
+import {dict} from '../../../../src/utils/object';
+import {mount} from 'enzyme';
+
+describes.sandboxed('SocialShare preact component v1.0', {}, () => {
+  it('errors when the required "type" attribute is not provided', () => {
```

Comment:
Do we also want cases for when children are given, for pre configured and non configured providers?

Reply:
The behavior is the same in these two cases.  There is a test for when children are provided for a pre-configured provider.